src/wp-includes/comment.php

```diff
@@ -3285,7 +3285,7 @@ function clean_comment_cache( $ids ) {
 		do_action( 'clean_comment_cache', $id );
 	}
 
-	wp_cache_set( 'last_changed', microtime(), 'comment' );
+	wp_cache_set_comments_last_changed();
```

Comment:
Sorry, I think I confused you posting the diff while discussing multiple tickets. I was thinking this would be good for the other PR for introducing the query groups.

Reply:
I noticed. I reverted the commit.